docs/general/polkadot-v1.md

```diff
@@ -142,7 +142,7 @@ further improved such that the tasks which are currently managed on the relay ch
 balances transfers, staking, and governance, can be delegated to
 [system parachains](../learn/learn-system-chains.md) to increase flexibility and to focus the use of
 the relay-chain to provide shared security and interoperability. Blockspace is only accessible
-through slot auctions, but an auction winner has access to a "freighter of blocks" regardless it is
+through auctions, but an auction winner has access to a "freighter of blocks" regardless it is
```

Comment:
Looking at the wording in this doc on Polkadot v1, it makes sense to talk about auctions and crowdloans in past tense.

Reply:
will go through that page in more detail and check